src/views/main.py

```diff
@@ -0,0 +1,188 @@
+from flask import render_template, request, redirect, url_for
+from src import app, mongo
+from src.datatable import DataTablesServer
+from flask_login import login_required, current_user
+from bson import ObjectId
+from functools import wraps
+import json
+
+
+# make connection with MongoDB data
+variant_col = mongo.db.variant
+user_col = mongo.db.user
+
+columns = ['#', '_id', 'total']
+default_fields = ["Details", "dn_no", "gene", "Position", "inheritanceMode", "cdna", "protein", "effect", "ref", 
+                  "genotype Patient", "genotype Mother", "genotype Father", "inheritedFrom", "GnomAD", "fullgnomen",
+                  "transcript"]
+default_order = {"dn_no": 1, "gene": 1}
+variants = []
+
+
+def login_required(f):
+    '''Wrapper around the login_required() function of Flask, to also check the active status of the user'''
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        if current_user.is_authenticated is False or check_if_user_active(current_user):
+            return redirect(url_for('account'))
+        return f(*args, **kwargs)
+    return decorated_function
```

Comment:
This function name is the same as one of your imports from flask_login. Either rename the function or use import `as` statement

Reply:
it is meant to be the same name with the @wraps. you change the imported function